indico/modules/events/contributions/forms.py

```diff
@@ -34,6 +34,7 @@
 
 
 class ContributionForm(IndicoForm):
+    _submitter_editable_fields = ('title', 'description', 'duration', 'person_link_data')
```

Comment:
Do we really want the duration to be editable? This is a mess once things are scheduled...

cc @plourenco

Reply:
Yes, was wondering about the same. Not needed, JACoW also confirmed.